docs/en/adfs.md

```diff
@@ -109,7 +109,11 @@ Repeat the same "Add Rule" as done above and select "Send Claims Using a Custom
 
 ## Set the secure hash algorithm
 
-By default ADFS uses a hash algorithm incompatible with *silverstripe-activedirectory* SAML implementation. You will need to change it from SHA-256 to SHA-1.
+By default ADFS uses SHA-256 for signing the requests, the Active Directory module by
+default uses this hash algorithm, but can be changed to use the less secure SHA-1.
```

Comment:
I'm not sure we even want to mention it can be changed to the "less secure" - when would anyone want to do that?